tdrs-backend/tdpservice/users/test/test_api/test_basic.py

```diff
@@ -37,11 +29,3 @@ def test_cannot_update_user_anonymously(api_client, user):
     """Test an unauthenticated user cannot update a user."""
     response = api_client.patch(f"/v1/users/{user.pk}/", {"first_name": "Jane"})
     assert response.status_code == status.HTTP_403_FORBIDDEN
-
-
-@pytest.mark.django_db
-def test_create_user(api_client, user_data):
-    """Test user creation."""
-    response = api_client.post("/v1/users/", user_data)
-    assert response.status_code == status.HTTP_201_CREATED
-    assert User.objects.filter(username=user_data["username"]).exists()
```

Comment:
@riatzukiza Instead of removing this test, can we instead assert that the status_code is a 404? (It might be 405 because it's a post, I don't recall)